README.md

```diff
@@ -74,7 +74,13 @@ Another good use case is if you have some periodic tasks to calculate daily stat
 
 ## Usage
 
-First, include the `CreatedId` module into your models. Note that any model you wish to include this module in must have a numeric primary key and a `created_at` columm.
+Run the generator to create the database table
+
+```
+  rails generate created_id:install
```

Comment:
You don't need to create a generator for engine migrations. The engine detects the migrations and creates a task to copy the migrations to the main application.

```bash
rails created_id_engine:install:migrations
```

Reply:
Readme updated